src/generic_layout.js

```diff
@@ -70,6 +70,11 @@ class GenericLayout {
             this.appendCreateVideoRequests(this.slide.videos, requests);
         }
 
+        if (this.slide.notes && this.slide.notes.rawText.length) {
```

Comment:
Mostly it was to avoid having to update a bunch of unrelated unit tests. Anyway, changed the extraction logic so that notes is null by default and only present when there are actual notes. This check is simplified to just an existence check.